src/components/atomic/TextInput/TextInput.scss

```diff
@@ -13,7 +13,7 @@
 	&[type='url'],
 	&[type='week'] {
 		background-color: white;
-		margin: 0;
+		margin-bottom: 20	px;
```

Comment:
weird spacing going on here

Reply:
I'll go ahead and fix that, not sure why there's so much spacing